project/JdkOptions.scala

```diff
@@ -26,6 +26,16 @@ object JdkOptions extends AutoPlugin {
     VersionNumber(specificationVersion).matchesSemVer(SemanticSelector(s"=1.8"))
   val isJdk11orHigher: Boolean =
     VersionNumber(specificationVersion).matchesSemVer(SemanticSelector(">=11"))
+  val isJdk17orHigher: Boolean =
+    VersionNumber(specificationVersion).matchesSemVer(SemanticSelector(">=17"))
+
+  val versionSpecificJavaOptions =
+    if (isJdk17orHigher) {
+      // for aeron
+      "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED" ::
```

Comment:
we should add a note about this in the docs

Reply:
I created an issue for that already #31087